ggml-rs/build.rs

```diff
@@ -88,6 +89,16 @@ fn main() {
     }
     build.warnings(false);
     build.compile("ggml");
+
+    let header_path = "./ggml/include/ggml/ggml.h";
```

Comment:
Ah yeah - we originally did this, but moved to pre-generated bindings because not everyone has Clang/LLVM available. (#22)

Unfortunately, I think we'll have to stick with the pre-generated bindings 😦

Reply:
That...is unfortunate 😅